protostar/commands/test/test_cases.py

```diff
@@ -69,12 +69,8 @@ def display(self, include_stdout_section: bool = False) -> str:
                     )
 
         stdout_elements: List[str] = []
-        if (
-            self.captured_test_stdout or self.captured_setup_stdout
-        ) and include_stdout_section:
-            stdout_elements = _get_formatted_stdout(
-                self.captured_test_stdout, self.captured_setup_stdout, "GREEN"
-            )
+        if include_stdout_section:
```

Comment:
given that we have removed the CLI flag, I think this if is unnecessary now? could you please remove it?

Reply:
I think that keeping this in here might be a good idea as it makes implementing printing patterns easier in the future, wdyt? It's not a big overhead imo.

Reply:
I'm not so inclined actually to say that this is going to be future-proof, hence I suggest treating this as dead code.